account_move_line_stock_info/hooks.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Carlos Lopez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def post_init_hook(cr, registry):
+
+    """ INIT stock_move_id references in acount move line from account_move"""
+    # FOR stock moves
+    cr.execute(
+        """
+        UPDATE account_move_line aml
```

Comment:
@AaronHForgeFlow @celm1990 My question looking at this query: Why the stock_move_id field is not a related on account_move one ?

Reply:
This module was created in v8 and Odoo in that version Odoo was not adding the stock_move_id in the account move I believe

When Odoo added that field I think that this module could be adapted to that (making it related stored), but in any case I think it is still consistent. It uses the prepare_account_move_line method:https://github.com/odoo/odoo/blob/13.0/addons/stock_account/models/stock_move.py#L375 which it is always called form the same stock move that is used to fill the information in the account_move. 
Perhaps in newer versions this could be adapted.